ecommerce/baskets/models.py

```diff
@@ -0,0 +1,56 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from baskets import enums
+from core.models import BaseAbstractModel
+
+from customers.models import Customer
+from products.models import Product
+
+
+class Basket(BaseAbstractModel):
+    customer = models.ForeignKey(Customer,
+                                 verbose_name=_("Customer"),
+                                 on_delete=models.CASCADE,
+                                 null=True
+                                 )
+
+    status = models.CharField(choices=enums.Status.choices,
+                              verbose_name=_("Status"),
+                              max_length=15
+                              )
+
+    class Meta:
+        verbose_name = _("basket")
+        verbose_name_plural = _("baskets")
+
+    def __str__(self):
+        return f"{self.customer} - {self.status}"
```

Comment:
you can use self.get_status_display()
https://docs.djangoproject.com/en/3.2/ref/models/instances/#django.db.models.Model.get_FOO_display